src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,125 @@
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  choosedMovie: Dispatch<SetStateAction<Movie[]>>,
+  movies: Movie[],
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ choosedMovie, movies }) => {
+  const [title, setTitle] = useState('');
+  const [preview, setPreview] = useState<Movie | null>(null);
+  const [error, setError] = useState(false);
+  const [duplicate, setDuplicate] = useState(false);
+
+  return (
+    <>
+      <form
+        className="find-movie"
+      >
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={classnames(
+                'input',
+                {
+                  'is-danger': error,
+                },
+              )}
+              value={title}
+              onChange={(event) => {
+                setTitle(event.target.value);
+                setError(false);
+                setDuplicate(false);
+              }}
+            />
+          </div>
+
+          {error
+          && (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
+              data-cy="find"
+              className="button is-light"
+              disabled={!title}
+              onClick={() => {
+                getFilmByTitle(title)
+                  .then(film => {
+                    if (film.Error) {
+                      setError(true);
+                    } else {
+                      return setPreview(film);
+                    }
+
+                    return null;
+                  });
+              }}
+            >
+              Find a movie
+            </button>
+          </div>
+
+          <div className="control">
+            <button
+              type="button"
+              data-cy="add"
+              className="button is-primary"
+              disabled={error}
+              onClick={() => {
+                if (preview !== null
```

Comment:
```suggestion
                if (preview
```

Reply:
If I'm not mistaken, my code means the same as yours, but it looks more concise😃